multipaz-facedetect/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
I think this should be called `multipaz-face-detection`, not `multipaz-facedetect`.

Also, I don't think there's a need for a `.gitignore` file in this directory, none of the other libraries have this and the top-level one should cover this build directory.